presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -1060,9 +1060,9 @@ public void testViewColumnAccessControl()
                 privilege("orders", CREATE_VIEW_WITH_SELECT_COLUMNS));
 
         // verify selecting from a view over a table requires the view owner to have special view creation privileges for the table
-        assertAccessDenied(
+        assertWrappedAccessDenied(
                 "SELECT * FROM " + columnAccessViewName,
-                "View owner 'test_view_access_owner' cannot create view that selects from .*.orders.*",
+                "View owner does not have sufficient privileges: Access Denied: View owner 'test_view_access_owner' cannot create view that selects from \\w+.\\w+.orders\\w*",
```

Comment:
The full error message is:
```
Access Denied: View owner does not have sufficient privileges: Access Denied: View owner 'test_view_access_owner' cannot create view that selects from \\w+.\\w+.orders\
```

There are two issues here.
 - `Access Denied: ` is unnecessarly doubled
 - error message is long and detailed (but on the hand it could be useful to understand why just happened). How about raising only `Access Denied: View owner does not have sufficient privileges` without the second part? @electrum @findepi thoughts?

Reply:
assertWrappedAccessDenied checks the full message ("View owner does not have sufficient privileges: Access Denied: View owner 'test_view_access_owner' cannot create view that selects from \\w+.\\w+.orders\\w*")
1. already resolved here https://github.com/prestosql/presto/pull/5086#discussion_r489786857

Reply:
1. Oh... I get it now.

I am not sure if it is nice looking error message:
`View owner does not have sufficient privileges: Access Denied: View owner 'test_view_access_owner' cannot create view `
vs
`
Access Denied: View owner does not have sufficient privileges: View owner 'test_view_access_owner' cannot create view`
vs
`Access Denied: View owner does not have sufficient privileges`

I vote for the last one. If one wants more details then in CLI can use `--debug` in web ui can see the stack trace in web ui.

Reply:
The example is... well, an example, right?
The second part can be many different things, so it may be less or more "boring".

My point is -- I don't want to be faced with "Access Denied: View owner does not have sufficient privileges" (period) error message and be asked by the user "ok, so what do i need to change?". It user's query is a big one, it may take a while until you learn which view, which view owner, and what permission they are missing.

Reply:
Ok, let it be `Access Denied: View owner does not have sufficient privileges: View owner 'test_view_access_owner' cannot create view`.

Reply:
done.